src/output/plugins/OSXOutputPlugin.cxx

```diff
@@ -76,6 +74,7 @@ struct OSXOutput final : AudioOutput {
 	 */
 	bool dop_setting;
 	bool dop_enabled;
+	std::atomic<bool> cancel;
```

Comment:
This requires `<atomic>`; also, `std::atomic_bool` is a shortcut which might be useful here.

Reply:
Done in https://github.com/MusicPlayerDaemon/MPD/pull/598